eureka2-external-server/src/main/java/com/netflix/eureka2/server/EurekaClientProvider.java

```diff
@@ -69,7 +69,6 @@ public Integer call(ServerBootstrap server) {
                 }
         );
         return Eureka.newClientBuilder(discoveryResolver, registrationResolver)
-                .withCodec(config.getCodec())
```

Comment:
Can we still run EurekaRead server with arbitrary codec? JSON is very useful for debugging purposes.